C/plugins/storage/sqlite/plugin.cpp

```diff
@@ -40,8 +41,37 @@ const char *default_config = QUOTE({
 			"default" : "5",
 			"displayName" : "Pool Size",
 			"order" : "1"
-			}
-		});
+		},
+		"nReadingsPerDb" : {
+			"description" : "Number of readings tables per  database",
+			"type" : "integer",
+			"default" : "15",
+			"displayName" : "N Readings per database",
+			"order" : "2"
+		},
+		"nDbPreallocate" : {
+			"description" : "Number of databases to allocate in advance, NOTE: SQLite has a maximum number of attachable databases by default at 10",
```

Comment:
"Number of databases to allocate in advance. NOTE: SQLite has a default maximum of 10 attachable databases"

Reply:
done